dockerfiles/init/modules/openshift/files/scripts/che-config-map

```diff
@@ -0,0 +1,34 @@
+# To put some extra CHE server configurations just add here CHE ENV vars following format:
+# KEY: "value"
+CHE_API: ${HTTP_PROTOCOL}://che-${OPENSHIFT_NAMESPACE_URL}/api
+CHE_DEBUG_SERVER: ${CHE_DEBUG_SERVER}
+CHE_HOST: che-${OPENSHIFT_NAMESPACE_URL}
+CHE_INFRASTRUCTURE_ACTIVE: openshift
+CHE_INFRA_OPENSHIFT_BOOTSTRAPPER_BINARY__URL: ${HTTP_PROTOCOL}://che-${OPENSHIFT_NAMESPACE_URL}/agent-binaries/linux_amd64/bootstrapper/bootstrapper
+CHE_INFRA_OPENSHIFT_MACHINE__START__TIMEOUT__MIN: "5"
+CHE_INFRA_OPENSHIFT_MASTER__URL: ""
```

Comment:
Let's set value as env var to make it possible to override it

Reply:
which exactly value?

Reply:
I'm talking about
`CHE_INFRA_OPENSHIFT_MASTER__URL: ${CHE_INFRA_OPENSHIFT_MASTER__URL}`

As far as I understand it will be empty if `CHE_INFRA_OPENSHIFT_MASTER__URL` env var is missed, isn't it?